libcxx/test/std/containers/views/mdspan/extents/ctad.pass.cpp

```diff
@@ -13,10 +13,12 @@
 // explicit extents(Integrals...) -> see below;
 //   Constraints: (is_convertible_v<Integrals, size_t> && ...) is true.
 //
-// Remarks: The deduced type is dextents<size_t, sizeof...(Integrals)>.
+// Remarks: The deduced type is dextents<size_t, sizeof...(Integrals)>.           // until C++26
+// Remarks: The deduced type is extents<size_t, maybe-static-ext<Integrals>...>.  // since C++26
```

Comment:
```suggestion
// Remarks: The deduced type is extents<size_t, sizeof...(Integrals)>.            // until C++26
// Remarks: The deduced type is extents<size_t, maybe-static-ext<Integrals>...>.  // since C++26
```

Reply:
I think the deduced type can be written as either `dextents<size_t, sizeof...(Integrals)>` or `extents<size_t, dynamic_extent, dynamic_extent, ..., dynamic_extent>`, where `dynamic_extent` is repeated a total of `sizeof...(Integrals)` times.

Reply:
Sorry I missed this comment. In the wording we prefer to use the exact wording in the Standard. In the code, where allowed, we sometimes deviate.

Reply:
Thanks! I believe the existing patch matches the wording in the standard: https://github.com/cplusplus/draft/pull/6907/files#diff-3b5851f7056b7c68cb3ba2ab51cf0b75a780c6a11e5e181770067700454ace7bL19461-R19468
(Correct me if I'm wrong)